scripts/start.mjs

```diff
@@ -0,0 +1,35 @@
+import { spawn, exec } from "child_process";
+console.log("Starting a hardhat node...");
+const hardhat = spawn("yarn contracts hardhat node", { shell: true });
+
+hardhat.stdout.on("data", (data) => {
```

Comment:
Instead of parsing the hardhat node output, it might be better to poll `http://localhost:8545` for connectivity (e.g. using `eth_getChainId`).

Reply:
This can be done at the top level using async commands:

```js
for (;;) {
  await fetch (`http://localhost:8545`, { method: 'POST' })
    .catch(() => {})
  await new Promise(r => setTimeout(r, 1000))
}
```